### DISCUSSION TITLE
[Feature request] LXQt minor point release before Lubuntu hits Feature Freeze

### DISCUSSION THREAD

**Discussion by tsimonq2** [lxqt/lxqt#2250] [Q&A] [answered]
They say you don't know the answer to a question unless you ask. I'm okay with no as an answer. Figured I'd ask though.

I'm seeing a few useful upstream commits that I'd like to include in Lubuntu 22.10. I was wondering if we could do a point release [before Feature Freeze](https://discourse.ubuntu.com/t/kinetic-kudu-release-schedule/27263) so I don't have to maintain a local patchset.

Thanks for your consideration.

**Comment by tsujan** [comment, marked as answer]
We agreed that point releases of components should be made only for critical fixes or important changes (https://github.com/lxqt/libqtxdg/releases/tag/3.9.1 is an example of the latter). So, most probably, there will be no point release for LXQt 1.1.0.

BTW, I converted this page to a discussion. It's hardly a feature request.

**Comment by tsimonq2** [comment]
Thank you for your consideration and response, and no problem. If anything changes, I'd be happy to help.
